docs/content.zh/docs/learn-flink/etl.md

```diff
@@ -156,56 +154,50 @@ keyBy(ride -> GeoUtils.mapToGridCell(ride.startLon, ride.startLat))
 
 ### Keyed Stream 的聚合
 
-以下代码为每个行程结束事件创建了一个新的包含 `startCell` 和时长（分钟）的元组流：
+以下代码为每个行程结束事件创建了一个新的包含 `startCell` 和距离的元组流：
 
 ```java
-import org.joda.time.Interval;
-
-DataStream<Tuple2<Integer, Minutes>> minutesByStartCell = enrichedNYCRides
-    .flatMap(new FlatMapFunction<EnrichedRide, Tuple2<Integer, Minutes>>() {
+DataStream<Tuple2<Integer, Double>> minutesByStartCell = enrichedNYCRides
```

Comment:
name should be distanceByStartCell

Reply:
Thanks for pointing it out. I will fix it now.